plugins/shopify/src/editors/ShopifyProductPicker.tsx

```diff
@@ -258,7 +258,7 @@ export class ShopifyProductPicker extends SafeComponent<ShopifyProductPickerProp
     return {
       '@type': '@builder.io/core:Request',
       request: {
-        url: `${appState.config.apiRoot()}/api/v1/shopify/products/{{this.options.product}}.json?apiKey=${
+        url: `${appState.config.apiRoot()}/api/v1/shopify/products/${productId}.json?apiKey=${
```

Comment:
Same thing here for the `options` object

Reply:
it's used by the plugin to identify the current value